database.sql

```diff
@@ -1,9 +1,37 @@
 -- ------------------------------------------
 -- Friendica 2020.06-dev (Red Hot Poker)
--- DB_UPDATE_VERSION 1348
+-- DB_UPDATE_VERSION 1350
 -- ------------------------------------------
 
 
+--
+-- TABLE gserver
+--
+CREATE TABLE IF NOT EXISTS `gserver` (
```

Comment:
Can you please move back this definition in the table name alphabetical order it was in before?

Reply:
Sadly this is not possible. Due to the foreign keys, the tables where foreign keys rely upon needs to be created first.

Reply:
Got it.